app/lib/project/ImportController.js

```diff
@@ -75,8 +76,11 @@ define([
                     return true;
                 }.bind( null, this._master )
             };
-            this._sourceGlyphSet = this._project.getGlyphSet(
-                        false, this._sourceUFODir, undefined, options);
+
+            console.log("maybe here?")
+            this._sourceGlyphSet = GlyphSet.factory(
+                    false, this._io, this._sourceUFODir + "/glyphs", undefined, /*UFOVersion*/ 2 /*FIXME: this is a hardcoded value!*/, options);
```

Comment:
The fix for not hardcoding the UFO version is most probably to finish the port of ufoJS.